### ISSUE TITLE
Add SC21 zenodo DOI

### ISSUE THREAD

**Issue by gflofst** [IO500/webpage#209]
https://zenodo.org/record/6462508#.YljCmovMKUk

Can you add this?

**Comment by gflofst** [comment]
Yes, I created ones for all lists after the reset (SC18). Let me get them to you.

Best,

Jay

From: Jean Luca Bez ***@***.***>
Sent: Monday, April 18, 2022 7:37 PM
To: IO500/webpage ***@***.***>
Cc: Lofstead, Gerald F II ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [IO500/webpage] Add SC21 zenodo DOI (Issue #209)


@gflofst<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgflofst&data=04%7C01%7Cgflofst%40sandia.gov%7Cc8fa794b5b8e416e74a508da21a513f1%7C7ccb5a20a303498cb0c129007381b574%7C1%7C0%7C637859290147576225%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=9acHcEpo%2BKMvwbFEZkJOqckM0Yu8eTylRanBF%2F9zSeQ%3D&reserved=0> do we have for previous years too?

-

**Comment by jeanbez** [comment]
Fixed in https://github.com/IO500/webpage/pull/212
